src/explore-education-statistics-admin/src/pages/release/data/components/DataFileDetailsTable.tsx

```diff
@@ -113,33 +113,27 @@ const DataFileDetailsTable = ({
         <tr>
           <th scope="row">Data file size</th>
           <td data-testid="Data file size">
-            {!dataFile.isQueuedZipUpload && (
-              <>
-                {dataFile.fileSize.size.toLocaleString()}{' '}
-                {dataFile.fileSize.unit}
-              </>
-            )}
+            <>
+              {dataFile.fileSize.size.toLocaleString()} {dataFile.fileSize.unit}
+            </>
           </td>
           {replacementDataFile && (
             <td data-testid="Replacement Data file size">
-              {!replacementDataFile.isQueuedZipUpload && (
-                <>
-                  {replacementDataFile.fileSize.size.toLocaleString()}{' '}
-                  {replacementDataFile.fileSize.unit}
-                </>
-              )}
+              <>
+                {replacementDataFile.fileSize.size.toLocaleString()}{' '}
+                {replacementDataFile.fileSize.unit}
+              </>
             </td>
           )}
         </tr>
         <tr>
           <th scope="row">Number of rows</th>
           <td data-testid="Number of rows">
-            {!dataFile.isQueuedZipUpload && dataFile.rows.toLocaleString()}
+            {dataFile.rows?.toLocaleString()}
```

Comment:
Not a biggy, but if there are no detected rows, we could show something like 'Unknown'?

Reply:
Thanks, made this change